gateway/1.6/modules/ROOT/pages/_partials/task-config-disable-logs.adoc

```diff
@@ -0,0 +1,158 @@
+//tag::config-disable-logs-intro[]
+
+Flex Gateway logging provides insights into startup, shutdown, API requests, API responses, policies, and Fluent Bit. Logging allows developers and administrators to identify and address any issues or anomalies. However, disabling log output might become necessary. Reasons include:
+
+* Compliance: Certain regulatory or compliance frameworks may impose restrictions on storing or retaining logs beyond a specific period. Disabling logs ensures compliance with such requirements and avoids unnecessary data retention.
+
+* Security: In some cases, logging sensitive information might pose a security risk. Disabling logs can prevent the storage of sensitive data, reducing the chances of potential data breaches.
```

Comment:
I think that the security risk is about sending the logs to Anypoint

Reply:
Hi @seblaz, is the concern that we're sending sensitive information to Anypoint, and that there can be data breaches when sending this information? If so, should we really be documenting that Anypoint is insecure?

Reply:
When I wrote this comment I thought about security risk involving outbound connections. Nevertheless, giving it a second thought, I think your comment is better.

IMHO, I would try to say something like this:
```suggestion
* Security: In some cases, logging sensitive information might pose a security risk. Not sending logs to a third party system can prevent the storage of sensitive data, reducing the chances of potential data breaches.
```

I think that customers are usually concerned about sending sensitive information to other systems that they don't own

Reply:
For now, I removed discussion about security.